src/components/LayoutWithRouting.tsx

```diff
@@ -19,6 +27,7 @@ const Route: React.FC<RouteProps> = ({ Component, ...pathProps }) => {
         navigate('/')
       }}
     >
+      <ScrollToTop />
```

Comment:
Do we need that in every `Route` component? Wouldn't just one instance in `LayoutWithRouting` (under `Router`) work?

Reply:
I don't think that would work, the `Router` component will only render one child, while we want to render this on every route. I agree it is redundant to have it on every route, so I'm happy to change it if we come up with a more elegant solution.

Reply:
Ah, right, I forgot we're using `@reach/router`, we could do that in `react-router` I think. I found this approach: https://github.com/reach/router/issues/262#issuecomment-484487915 I think it's better than having that in each route, and later we can get rid of that once we move to `react-router`